packages/cadl-typescript/src/transform/transform.ts

```diff
@@ -69,7 +73,7 @@ export async function transformRLCModel(program: Program, emitterOptions: RLCOpt
 
 function transformApiVersionParam(program: Program): Parameter | undefined {
   const apiVersion = getServiceVersion(program);
-  if (apiVersion) {
+  if (apiVersion && apiVersion !== "0000-00-00") {
```

Comment:
why add this filter `apiVersion !== "0000-00-00"` ?

Reply:
Just wondering if you think the `0000-00-00` is not a valid apiVersion, could we move the logic to method `getServiceVersion`?

```
if(apiVersion !== "0000-00-00") {
 return undefine;
}
```

Reply:
Yeah it looks like if the CADL definition doesn't define a service version, the compiler gives us `"0000-00-00"` I'll move this to `getServiceVersion` and bring this up to the CADL team as it may be an issue in the compiler